interfaces/builtin/fuse_support.go

```diff
@@ -72,11 +76,14 @@ deny /etc/fuse.conf r,
 `
 
 func init() {
+	// Ubuntu 14.04 does not support the fuse-support interface.
 	registerIface(&commonInterface{
 		name:                  "fuse-support",
 		summary:               fuseSupportSummary,
+		implicitOnCore:        true,
+		implicitOnClassic:     !(release.ReleaseInfo.ID == "ubuntu" && release.ReleaseInfo.VersionID == "14.04"),
```

Comment:
This is not an unrelated change. It used to be defined exactly this way in `implicit.go` before.